packages/api/src/hooks/useCryptoTransactions.ts

```diff
@@ -58,6 +58,45 @@ const getStatusName = (status_code: TModifiedTransaction['status_code']) => {
     }
 };
 
+const getStatusDescription = (
+    transaction_type: TModifiedTransaction['transaction_type'],
+    status_code: TModifiedTransaction['status_code']
+) => {
+    switch (status_code) {
+        // deposit-specific:
+        case 'CONFIRMED':
+            return 'Your deposit is successful.';
+        case 'PENDING':
+            return "We've received your request and are waiting for more blockchain confirmations.";
```

Comment:
I am constantly wondering - is our translation system capable of handling keys like
"wallets.transations.states.CONFIRMED"
"wallets.transations.states.PENDING"
"wallets.transations.states.CANCELLED"
... etc etc

That would enable devs to only use enums and completely get rid of such strings from codebase and never ever write such switch-case stataments anymore :D

Reply:
that would also completely remove discussion "where does status description belongs? is it a hook concern, or component concern?"